bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -559,6 +559,7 @@ static function (Task $task) {
             DataObject\Service::enrichLayoutDefinition($objectData['layout'], $object);
             $eventDispatcher->dispatch($event, AdminEvents::OBJECT_GET_PRE_SEND_DATA);
```

Comment:
Just wondering what happens if the event listener tries to find a 'child' node. If I understand it correctly $objectData['layout'] uses 'children' nodes from now on.

Reply:
@weisswurstkanone Should be working because of the new __get/__set methods in models/Element/ChildsCompatibilityTrait.php
https://github.com/pimcore/pimcore/blob/840f4bdeefc6aed2f443158cff654a7027ade3e5/models/Element/ChildsCompatibilityTrait.php